data/domain/world-6/farming.tsx

```diff
@@ -17,6 +17,104 @@ import { JadeUpgrade, PristineCharm, Sneaking } from "./sneaking";
+        this.upgrades.push(new LandRankUpgrade(7,"Overgrowth Boost","Increases chance for Overgrowth by +{%",120,180));
+        this.upgrades.push(new LandRankUpgrade(8,"Production Megaboost","Increases the amount of crops harvested by +{%",100,250));
+        this.upgrades.push(new LandRankUpgrade(9,"Seed of Loot","Increases the Drop Rarity of all characters by +{%",10,400));
+        this.upgrades.push(new LandRankUpgrade(10,"Evolution Superboost","Increases next crop chance by +{% multiplicatively!",3000,500));
+        this.upgrades.push(new LandRankUpgrade(11,"Overgrowth Megaboost","Increases chance for Overgrowth by +{%",340,600));
+        this.upgrades.push(new LandRankUpgrade(12,"Farmtastic Megaboost","Increases Farming Skill EXP gained by +{%",110,700));
+        this.upgrades.push(new LandRankUpgrade(13,"Soil Exp Superboost","All plots of land gain +{% more Rank EXP",520,900));
+        this.upgrades.push(new LandRankUpgrade(14,"Seed of Damage","Gives a +{% Total Damage bonus to all characters",20,1200));
+        this.upgrades.push(new LandRankUpgrade(15,"Evolution Ultraboost","Increases next crop chance by +{% multiplicatively!",40000,1300));
+        this.upgrades.push(new LandRankUpgrade(16,"Farmtastic Superboost","Increases Farming Skill EXP gained by +{%",220,1500));
+        this.upgrades.push(new LandRankUpgrade(17,"Production Superboost","Increases the amount of crops harvested by +{%",600,1750));
+        this.upgrades.push(new LandRankUpgrade(18,"Overgrowth Superboost","Increases chance for Overgrowth by +{%",1500,2000));
+        this.upgrades.push(new LandRankUpgrade(19,"Seed of Stats","Gives a +{% All Stat bonus to your characters",5,3500));
+    }
+
+    // This function send the sum of all bonus from a same type, Lava use this in his code so having it here helps to "translate" his code into IE code
+    getTotalUpgradeBonusForBonus = (bonusType: LandRankBonusType): number => {
+        switch(bonusType) {
+            case LandRankBonusType.NextCropChance:
+                return (1 + this.getUpgradeBonusFromIndex(3) / 100) * (1 + this.getUpgradeBonusFromIndex(10) / 100) * (1 + this.getUpgradeBonusFromIndex(15) / 100);
+            case LandRankBonusType.AmountHarvested:
+                return this.getUpgradeBonusFromIndex(8) + this.getUpgradeBonusFromIndex(17);
+            case LandRankBonusType.LandRankEXP:
+                return this.getUpgradeBonusFromIndex(6) + this.getUpgradeBonusFromIndex(13);
+            case LandRankBonusType.OGChance:
+                return this.getUpgradeBonusFromIndex(7) + (this.getUpgradeBonusFromIndex(11) + this.getUpgradeBonusFromIndex(18));
+            case LandRankBonusType.FarmingSkillExp:
+                return this.getUpgradeBonusFromIndex(5) + (this.getUpgradeBonusFromIndex(12) + this.getUpgradeBonusFromIndex(16));
+            default: return 1;
+        }
+    }
+
+    getUpgradeBonusFromIndex = (index: number): number => {
```

Comment:
Super nitpicky 
```suggestion
    getUpgradeBonusByIndex = (index: number): number => {
```

Reply:
Might be nitpicky but that's worth changing for consistence on the site